docs/integrations/neutron.mdx

```diff
@@ -68,39 +68,39 @@ title: Neutron
 
 A comprehensive overview of the relevant metrics is hosted in the root [metrics.md](https://github.com/skip-mev/slinky/blob/262fddc8ff1335c87886cf7aacce2efea8164246/metrics.md).
 
-A further breakdown of metrics are listed in the Slinky repo, categorized by subsystem:
+A further breakdown of metrics are listed in the Connect repo, categorized by subsystem:
 
 - [Application-Side Metrics](https://github.com/skip-mev/slinky/blob/a6577aa779f1f98c47ec95d626e4af70949af7b1/service/metrics/README.md)
 - [General Oracle Sidecar Metrics](https://github.com/skip-mev/slinky/blob/a6577aa779f1f98c47ec95d626e4af70949af7b1/oracle/metrics/README.md)
 - [Sidecar Provider Metrics](https://github.com/skip-mev/slinky/blob/a6577aa779f1f98c47ec95d626e4af70949af7b1/providers/base/metrics/README.md)
 - [API Provider Metrics](https://github.com/skip-mev/slinky/blob/a6577aa779f1f98c47ec95d626e4af70949af7b1/providers/base/api/metrics/README.md)
 - [Websocket Provider Metrics](https://github.com/skip-mev/slinky/blob/a6577aa779f1f98c47ec95d626e4af70949af7b1/providers/base/websocket/metrics/README.md)
 
-Additionally, the logs from your chain node binary will contain the following error if it is unable to connect to Slinky to grab prices:
+Additionally, the logs from your chain node binary will contain the following error if it is unable to connect to the sidecar to grab prices:
```

Comment:
if we want to be aligned across docs, we should probably figure out the standard here:
Slinky → Connect
Slinky sidecar → Connect sidecar???
Slinky sidecar → Connect instance???